package.json

```diff
@@ -17,15 +17,15 @@
     "node": ">= 10.0"
   },
   "scripts": {
-    "audit": "ep-node-build audit",
```

Comment:
Note: anyone who misses this will just run `npm audit` instead of `npm run audit`